kod/object/active/holder/room/razarm/razainn.kod

```diff
@@ -24,28 +24,26 @@ resources:
 
    razainn_Back_Door = "The back door is locked."
 
-   razainn_Signname = "Welcome"
+   razainn_Signname = "WASD to move, SPACE to open doors."
```

Comment:
Bravo lol

Number one complaint we got playtesting with non m59 players was "how the fuck do I open a door?"